test/tools/libtesteth/TestSuite.h

```diff
@@ -36,6 +36,7 @@ class TestSuite
 
 	// A folder of the test suite. like "VMTests". should be implemented for each test suite.
 	virtual std::string suiteFolder() const = 0;
```

Comment:
If `suiteFolder()` and `suiteFillerFolder()` are not used anywhere besides `runAllTestsInFolder()` & `copyAllTestsFromFolder()`, make them `protected`